scripts/postinstall.sh

```diff
@@ -1,3 +1,27 @@
 #!/bin/bash
 
-cp node_modules/wikipedia-preview/dist/wikipedia-preview.production.js assets/js
+# preparing compiled js and css file
+git submodule update
+cd wikipedia-preview
+git reset --hard
+npm i
+
+# compiled js
+sed -i "/import '.*\/style\/.*'/d" src/*.js
+sed -i "/import '.*\/style\/.*'/d" src/**/*.js
+npm run build
+
+# compiled css
+sed -i "s/inline-svg( '..\/images\/\(.*\)' )/url( \/wp-content\/plugins\/wikipediapreview-wordpress\/assets\/images\/\1 )/g" style/*.less
```

Comment:
This prints `sed: 1: "style/gallery.less": unterminated substitute pattern` and doesn't work when I run it. By "doesn't work" I mean the resulting file still has `inline-svg(...)`

Reply:
can you try it again with the latest change https://github.com/wikimedia/wikipediapreview-wordpress/pull/7/commits/628749dfec89f7447e758259bb531b3e5548d3f8

This serves as the quick fix, I'm at the same time working on the less custom plugin.

Reply:
Still not working... not worth hacking it further now that we have a good plan with config variables